src/components/TradeAsset/TradingForm.tsx

```diff
@@ -76,12 +77,14 @@ function TradingForm(props: Props) {
             </Box>
           ) : null}
           <div style={{ flexGrow: 1 }} />
-          <DialogActions
-            amount={amount}
-            disabled={amountString === "" || isDisabled(amount, price, Number.parseFloat(props.sellingBalance))}
-            price={price}
-            style={{ justifySelf: "flex-end" }}
-          />
+          {isSmallScreen ? null : (
```

Comment:
One detail I'd like to have fixed before merging this:

Let's deduplicate by putting the `<DialogActions ... />` in a variable we use in both places.